implementation/common/src/main/java/io/smallrye/jwt/util/KeyUtils.java

```diff
@@ -246,6 +249,42 @@ public static SecretKey createSecretKeyFromSecret(String secret) {
         return new SecretKeySpec(secretBytes, "AES");
     }
 
+    /**
+     * Generates a SecretKey.
+     *
+     * @param algo key encryption algorithm.
+     * @return SecretKey.
+     * @throws NoSuchAlgorithmException algorithm not found.
+     */
+    public static SecretKey generateSecretKey(KeyEncryptionAlgorithm algo) throws InvalidAlgorithmParameterException {
+        final String algoName = getSymmetricAlgoName(algo);
+
+        int keySizeBits = getKeySizeFromAlgo(algo);
```

Comment:
Please have an internal function which will use `AES` scheme - that will be fine - since it is not really used